content/reference/admin/private_locations/configuration/dedicated/index.md

```diff
@@ -0,0 +1,147 @@
+The control plane possesses a private key for establishing connections, while the corresponding public key is shared and configured on every host in the location.
+
+### Host prerequisites
+
+Location hosts environment must include:
+- Java runtime environment
+- `bash` the GNU Project's shell
+- `jq` a lightweight and flexible command-line JSON processor
+- `curl` command line tool for transferring data with URLs
+- `~/.ssh/authorized_keys` with control plane public key
+
+### Control plane configuration file
+
+```bash
+control-plane {
+  # Control plane token
+  token = "cpt_example_c7oze5djp3u14a5xqjanh..."
+  # Control plane token with an environment variable
+  token = ${?CONTROL_PLANE_TOKEN}
+  # Control plane description (optional)
+  description = "my control plane description"
+  # Locations configurations
+  locations = [
+    {
+      # Private location ID, must be prefixed by prl_, only consist of numbers 0-9, 
+      # lowercase letters a-z, and underscores, with a max length of 30 characters
+      id = "prl_private_location_example"
+      # Private location description (optional)
+      description = "Private Location on Dedicated Machines"
+      # Private location type
+      type = "dedicated"
+      ssh {
+        # SSH user
+        user = "gatling"
+        ## SSH private key
+        private-key {
+          # SSH private key path used to secure connections
+          path = "private/key/path"
+          ## SSH private key password (optional, default: none)
+          password = "password"
+           # SSH private key password with an environment variable
+          password = ${?PRIVATE_KEY_PASSWORD}
+        }
+        # SSH port (optional, default: 22)
+        # port = 22
+        # SSH connection timeout (optional, default 15 seconds)
+        # connection-timeout = 15 seconds
+      }
+      # Working directory on hosts, must be executable (optional, default: /tmp)
+      # working-directory = "/tmp
+      
+      # Hosts accessible with the SSH private key, on configured port (hostnames or IP addresses)
+      hosts = [
+        "localhost",
+        "0.0.0.0"
+      ]
+      
+      # Java configuration (following configuration properties are optional)
+      # System properties (optional)
+      system-properties {
+        # ExampleKey = ExampleValue
+      }
+      # Overwrite JAVA_HOME definition (optional)
+      # java-home = "/usr/lib/jvm/zulu"
+      # JVM Options (optional)
+      # Default ones, that can be overriden with precedence:
+      # [
+      #   "-Xmx4G", 
+      #   "-XX:MaxInlineLevel=20", 
+      #   "-XX:MaxTrivialSize=12", 
+      #   "-XX:+IgnoreUnrecognizedVMOptions", 
+      #   "--add-opens=java.base/java.nio=ALL-UNNAMED", 
+      #   "--add-opens=java.base/jdk.internal.misc=ALL-UNNAMED"
+      # ]
+      # Based on your instance configuration, you may want to update Xmx and Xms values.
+      # jvm-options = ["-Xmx4G", "-Xms512M"]
```

Comment:
```suggestion
      ## Overwrite JAVA_HOME definition (optional)
      # java-home = "/usr/lib/jvm/zulu"
      ## JVM Options (optional)
      ## Default ones, that can be overriden with precedence:
      ## [
      ##   "-Xmx4G", 
      ##   "-XX:MaxInlineLevel=20", 
      ##   "-XX:MaxTrivialSize=12", 
      ##   "-XX:+IgnoreUnrecognizedVMOptions", 
      ##   "--add-opens=java.base/java.nio=ALL-UNNAMED", 
      ##   "--add-opens=java.base/jdk.internal.misc=ALL-UNNAMED"
      ## ]
      ## Based on your instance configuration, you may want to update Xmx and Xms values.
      # jvm-options = ["-Xmx4G", "-Xms512M"]
```

But as the default values are the one commented by default...

```suggestion
      ## Overwrite JAVA_HOME definition (optional)
      # java-home = "/usr/lib/jvm/zulu"

      ## JVM Options (optional)
      ## Based on your instance configuration, you may want to update Xmx and Xms values.
      # jvm-options = [
      #   "-Xmx4G", 
      #   "-XX:MaxInlineLevel=20", 
      #   "-XX:MaxTrivialSize=12", 
      #   "-XX:+IgnoreUnrecognizedVMOptions", 
      #   "--add-opens=java.base/java.nio=ALL-UNNAMED", 
      #   "--add-opens=java.base/jdk.internal.misc=ALL-UNNAMED"
      # ]
```

Reply:
Maybe one day, but not today (must update all related documentations)